docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -550,6 +550,22 @@ The default is 30303. Ports must be [exposed appropriately](../Configuring-Panth
 !!!note
     This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#exposing-ports). 
 
+### upnp-enabled
+
+```bash tab="Syntax"
+--upnp-enabled
```

Comment:
Thinking forward to the other bounties (manual address, docker, kuberneties) are those modes compatible with upnp or mutually exclusive?  My understanding is that they will be mutually exclusive, so multiple flags for mutually exclusive modes creates a UX problem.

I think this should be replaced with a flag that takes an enum, like `--address-resolution=upnp` and the first two modes are `upnp` and `none`.  @jakehaugen will have stronger opinions about what the flag name should be, but using an enum instead of a flag for each should result in less error checking going forward.

Reply:
Yes, this makes sense to me. From doing some research, looks like "nat" is the go to word. After some discussion we landed on `--nat-method` as a good name. I'd say we should go with that unless anyone else has concerns.

Reply:
I like this -- changed in b19fc45.